tests/src/Functional/PersonContentRenderTest.php

```diff
@@ -0,0 +1,296 @@
+    $node = \Drupal::entityTypeManager()
+      ->getStorage('node')
+      ->create([
+        'type' => 'oe_sc_person',
+        'oe_sc_person_first_name' => 'Stefan',
+        'oe_sc_person_last_name' => 'Mayer',
+        'oe_sc_person_country' => 'DE',
+        'oe_sc_person_occupation' => 'DG Test',
+        'oe_sc_person_position' => 'Director',
+        'uid' => 1,
+        'status' => 1,
+      ]);
+    $node->save();
+    return $node;
+  }
+
+  /**
+   * Creates an example person node with required and optional fields.
+   *
+   * @return \Drupal\node\NodeInterface
+   *   Person node.
+   */
+  protected function createExamplePersonWithAllFields(): NodeInterface {
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = $this->createExamplePersonWithRequiredFieldsOnly();
+    // Create a sample image media entity to be embedded.
+    $image_file = File::create([
+      'uri' => $this->getTestFiles('image')[0]->uri,
+    ]);
+    $image_file->save();
+    $media_image = Media::create([
+      'bundle' => 'image',
+      'name' => 'Starter Image test',
+      'oe_media_image' => [
+        [
+          'target_id' => $image_file->id(),
+          'alt' => 'Starter Image test alt',
+          'title' => 'Starter Image test title',
+        ],
+      ],
+    ]);
+    $media_image->save();
+
+    // Create a sample document media entity to be embedded.
+    $document_file = File::create([
+      'uri' => $this->getTestFiles('text')[0]->uri,
+    ]);
+    $document_file->save();
+    $media_document = Media::create([
+      'bundle' => 'document',
+      'name' => 'Person document test',
+      'oe_media_file_type' => 'local',
+      'oe_media_file' => [
+        [
+          'target_id' => (int) $document_file->id(),
+          'alt' => 'Person document alt',
+          'title' => 'Person document title',
+        ],
+      ],
+    ]);
+    $media_document->save();
+
+    $document_reference = \Drupal::entityTypeManager()
+      ->getStorage('oe_document_reference')
+      ->create([
+        'type' => 'oe_document',
+        'oe_document' => $media_document,
+        'status' => 1,
+      ],
+      );
+    $document_reference->save();
+    $document_group_reference = \Drupal::entityTypeManager()
+      ->getStorage('oe_document_reference')
+      ->create([
+        'type' => 'oe_document_group',
+        'oe_title' => 'Curriculum Vitae',
+        'oe_documents' => [$media_document, $media_document],
+        'status' => 1,
+      ],
+      );
+    $document_reference->save();
+
+    $node->set('oe_summary', 'This field is used to add a short biography of the person.');
+    $node->set('oe_sc_person_additional_info', 'Additional information example field.');
+    $node->set('oe_social_media_links', [
+      'uri' => 'https://twitter.com',
+      'title' => 'Twitter profile',
+      'link_type' => 'twitter',
+    ]);
+    $node->set(
+      'oe_sc_person_documents',
+      [$document_reference, $document_group_reference]
```

Comment:
Note: I would put a trailing comma here, _but_ I see plenty of examples in core where they don't do this - probably to support PHP < 7.3.
Also phpcs does not complain with or without the trailing comma.
So let's keep it as is :)